src/server.c

```diff
@@ -431,7 +431,7 @@ main(int argc, char **argv) {
         if (endswith(info.iface, ".sock") || endswith(info.iface, ".socket")) {
 #if defined(LWS_USE_UNIX_SOCK) || defined(LWS_WITH_UNIX_SOCK)
             info.options |= LWS_SERVER_OPTION_UNIX_SOCK;
-            strncpy(server->socket_path, info.iface, sizeof(server->socket_path));
+            strncpy(server->socket_path, info.iface, sizeof(server->socket_path - 1));
```

Comment:
Shouldn't it be `sizeof(server->socket_path) - 1`?

Reply:
Sorry, that's embarrassing. Fixed..